sdk/elements/03-hydra-cli.md

```diff
@@ -316,4 +316,81 @@ Host: :host
 Public port: :your_port
 P2P port: :your_port
 Peer id: :peerId
-```
\ No newline at end of file
+```
+## Remote Monitoring
+
+We have introduced a tool in version `v0.10.0` that can monitor your metagraph and restart it if necessary.
+
+### Introduction
+This service monitors your metagraph and performs restarts as necessary. It is deployed using Ansible on a remote Ubuntu host. Commands such as `install-monitoring-service`, `remote-deploy-monitoring-service`, and `remote-start-monitoring-service` will be further explained in subsequent sections.
+
+The service is developed using `NodeJS`, and all necessary dependencies are installed on your remote instance during deployment.
+
+Running in the background with PM2, the service initiates checks at intervals specified in the configuration under the field: `check_healthy_interval_in_minutes`. It evaluates the health of the metagraph based on predefined and customizable `restart-conditions`, detailed in the [metagraph-monitoring-service](https://github.com/Constellation-Labs/metagraph-monitoring-service) repository. For example, if an unhealthy node is detected, the service triggers a restart.
+
+To restart a node or layer, the service first attempts to stop any running processes (referring to the layer). This operation requires sudo privileges without a password requirement (refer [to this](https://gcore.com/learning/how-to-disable-password-for-sudo-command/) document for instructions on setting up password-less sudo). In addition to terminating processes, log files from the node are moved to the `code/restart_logs` directory, which may also require sudo privileges.
+
+After these steps, the service restarts the node or layer and reintegrates it into the cluster.
```

Comment:
I think this section needs to zoom out and describe what the service does at a high level. I don't think we need to mention PM2 or sudo privileges here. What does the service do? 

Something like... 
The service runs health checks on a regular interval and evaluates the metagraph cluster on a set of predefined or developer-created health criteria (restart-conditions). Restart conditions can target the whole cluster, a specific layer, or individual nodes to ensure that the metagraph is operating properly in each case. If an issue is detected, the service uses SSH to access the impacted node(s) and restart their process and rejoin them to the network.